.test_mageck_mle/config/config.yml

```
@@ -0,0 +1,67 @@
+    l: 20 # shorten reads to l bases
+    extra: "" # Extra arguments for cutadapt
+  species: human
+
+csv: # If no fasta is available, provide a csv file with sgRNA sequences
+  name_column: 0 # Column number with sgRNA names
+  gene_column: 1 # Column number with gene names
+  sequence_column: 2 # Column number with sgRNA sequences
+
+mismatch: 0 # Mismatches allowed during alignment
+
+stats: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)